node/src/main/scala/coop/rchain/node/configuration/Configuration.scala

```diff
@@ -57,7 +57,9 @@ object Configuration {
   private val DefaultRequiredSigns              = 0
   private val DefaultApprovalProtocolDuration   = 5.minutes
   private val DefaultApprovalProtocolInterval   = 5.seconds
-  private val DefaultMaxMessageSize: Int        = 100 * 1024 * 1024
+  private val DefaultMaxMessageSize: Int        = 4 * 1024 * 1024
+  private val MaxMessageSizeMinimumValue: Int   = 1 * 1024 * 1024
+  private val MaxMessageSizeMaximumValue: Int   = 10 * 1024 * 1024
```

Comment:
Should be 2^24-1 or 16,777,215 octets like specified in the HTTP2 RFC 7540

Reply:
I made it smaller on purpose, you want ot make it 16 mb?